### PR TITLE
Publish cats scaladoc without the -cats suffix

### PR DESCRIPTION
**Pull request by rossabaker**
Fixes the 0.17 problems of #1061

### CODE REVIEW

**Comment by bfritz** [comment]
Project for some other day, but it would be nice if Travis published the hugo output somewhere like S3 so it's possible to review changes without building locally.

**Comment by bfritz** [comment]
@rossabaker Looks like there's another `-cats` [in Http4sGhPages.scala](https://github.com/rossabaker/http4s/blob/3f9c8feb48a1ce950bfbdeb3cf51e5736face823/project/Http4sGhPages.scala#L22).

**Comment by rossabaker** [comment]
💔 

I am apparently not good enough at grep to take on this ticket.

File: build.sbt [hunk 1]
```diff
@@ -319,15 +319,14 @@ lazy val docs = http4sProject("docs")
...
-
       IO.copyDirectory(
         source = preStageSiteDirectory.value,
         target = siteStageDirectory.value,
         overwrite = false,
         preserveLastModified = true)
       IO.copyDirectory(
         source = tutTargetDirectory.value,
-        target = siteStageDirectory.value / "content" / "v0.16-cats",
+        target = siteStageDirectory.value / "content" / "v${apiVersion.value._1}.${apiVersion.value._2}",
```
Comment (bfritz) [inline build.sbt]: Not sure if it's here or somewhere else, but when I `rm -rf docs/target` and then `sbt makeSite`, I end up with:
```
$ ls docs/target/hugo | grep apiver
vapiversion.value._1.apiversion.value._2/
```
Seems to go away with rossabaker/http4s#2 .

**Comment by ChristopherDavenport** [comment]
Lol. It's ok. I probably should have done it seeing that I wrote the commit. I thought by two places I intended `GHPages.scala` and missed the missed one on the same file.

**Comment by bfritz** [comment]
> I am apparently not good enough at grep to take on this ticket.

They're a pain to QA well.  Would be sweet if there was an easy way to see the output files in `docs/target` and see the results on a staging site.  Hopefully eliminating hacks in #972 will make it less prone to mistakes.

**Comment by ChristopherDavenport** [comment]
One More `build.sbt#L302`

**Comment by bfritz** [comment]
I think L302 is okay:
```
            if (version.value.endsWith("SNAPSHOT"))
              s"${s.browseUrl}/tree/cats€{FILE_PATH}.scala"
```
probably needs to be there until the `cats` branch becomes `master`.

**Comment by rossabaker** [comment]
And let us never speak of this PR again.

**Pull request merged**